CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ These are just guidelines, not rules, use your best judgment and feel free to pr
 
 ### Code of Conduct
 
-This project adheres to the [Contributor Covenant 1.2](http://contributor-covenant.org/version/1/2/0).
+This project adheres to the [Contributor Covenant 1.3](http://contributor-covenant.org/version/1/3/0).
```

Comment:
This link and the link in the README should probably link to the CODE_OF_CONDUCT document.